specification/peering/resource-manager/Microsoft.Peering/preview/2019-09-01-preview/peering.json

```diff
@@ -328,46 +336,521 @@
           }
         },
         "x-ms-examples": {
-          "List peer ASNs in a subscription": {
-            "$ref": "./examples/ListPeerAsnsBySubscription.json"
+          "List peer ASNs in a subscription": {
+            "$ref": "./examples/ListPeerAsnsBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/peeringLocations": {
+      "get": {
+        "tags": [
+          "PeeringLocations"
+        ],
+        "description": "Lists all of the available peering locations for the specified kind of peering.",
+        "operationId": "PeeringLocations_List",
+        "parameters": [
+          {
+            "name": "kind",
+            "in": "query",
+            "description": "The kind of the peering.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "Direct",
+              "Exchange"
+            ]
+          },
+          {
+            "name": "directPeeringType",
+            "in": "query",
+            "description": "The type of direct peering.",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "Edge",
+              "Transit",
+              "Cdn",
+              "Internal",
+              "Ix",
+              "IxRs"
```

Comment:
this is not modelled as a string. So, it is a breaking change to existing clients/SDK/CLIs...